docs: cleanup (#336)
bareos/bareos: docs/manuals/source/Appendix/ReleaseNotes.rst

Release Notes
=============

.. index:: Releases

.. warning::

>       While all the source code is published on `GitHub <https://github.com/bareos/bareos>`_, the releases of packages on http://download.bareos.org is limited to the initial versions of a major release. Later maintenance releases are only published on https://download.bareos.com.

Reviewer: ```suggestion
      While all the source code is published on `GitHub <https://github.com/bareos/bareos>`_, the releases of packages on http://download.bareos.org is limited to the initial versions of each major release. Later maintenance releases are only published on https://download.bareos.com.
```

This chapter concentrates on things to do when updating an existing Bareos installation.

> Please note, that some of the subreleases are only internal development releases.

Reviewer: ```suggestion
Please note, that some of the subreleases are only non official development releases.
```

Author: Is "non official" an/our official term for this? In doubt, internal/non official can be removed from the sentense at all.

> The overview about new feature of a release are shown at the :ref:`genindex` of this document.

Reviewer: ```suggestion
The overview about new features of a release are shown at the :ref:`genindex` of this document.
```
